src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/SeleniumTest.kt

```diff
@@ -38,20 +41,15 @@ abstract class SeleniumTest : CustomConfigTests()
         proxy.noProxy = "localhost"
         proxy.httpProxy = "localhost:" + hoverflyRule.proxyPort
         proxy.sslProxy = "localhost:" + hoverflyRule.proxyPort
-
+        System.setProperty("webdriver.chrome.whitelistedIps", "")
         driver = ChromeDriver(org.openqa.selenium.chrome.ChromeOptions()
                 .apply {
-                    addArguments("--ignore-certificate-errors", "--headless", "--no-sandbox")
+                    addArguments("--ignore-certificate-errors", "--headless", "--no-sandbox", "--disable-dev-shm-usage")
                     setProxy(proxy)
                 })
-        driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS)
+        driver.manage().timeouts().implicitlyWait(12, TimeUnit.SECONDS)
         wait = WebDriverWait(driver, 12)
-    }
-
-    @After
```

Comment:
this now handled by `DebugHelper` (the `finished` method there runs after this one, and needs the driver to still be alive)